tools/clang/test/CodeGenHLSL/quick-test/vecCombineSel.hlsl

```diff
@@ -0,0 +1,16 @@
+// RUN: %dxc -E main -T cs_6_0 %s | FileCheck %s
+
+
+
+// Make sure both channel is used.
```

Comment:
This is a clever (if roundabout) way to check for the ternary operation result. But it does the trick as a test case, so go for it.